tests/middleware/cds-execution.js

```diff
@@ -0,0 +1,148 @@
+import cdsExecution from "../../src/middleware/cds-execution";
+import * as types from "../../src/actions/action-types";
+import callServices from "../../src/retrieve-data-helpers/service-exchange";
+jest.mock("../../src/retrieve-data-helpers/service-exchange");
+
+describe("CDS Execution Middleware", () => {
```

Comment:
Can you follow the \<file\>.test.js naming convention?  Running "npm run test" doesn't pick this up in the test suite otherwise

Reply:
D'oh. Fixed :)